reporting_app/__init__.py

```diff
@@ -248,27 +257,37 @@ def report_sample(sample_id):
 
     return render_template(
         'sample_report.html',
+
         title=sample_id + ' Sample Report',
         tables=[
             datatable_cfg(
                 'Bioinformatics report for ' + sample_id,
                 'samples',
-                rest_api().api_url('aggregate/samples', match={'sample_id': sample_id}),
+                api_url=None,
```

Comment:
We should make api_url default to None, and check that we have either api_url or ajax_call